faunadb/client_test.go

```diff
@@ -1052,6 +1097,59 @@ func (s *ClientTestSuite) TestEvalTimeExpression() {
 	)
 }
 
+func (s *ClientTestSuite) TestEvalTimeAddExpression() {
+	var t time.Time
+
+	s.queryAndDecode(f.TimeAdd(f.Epoch(0, f.TimeUnitSecond), 1, f.TimeUnitHour), &t)
+	s.Require().Equal(t,
+		time.Unix(0, 0).UTC().
+			Add(time.Duration(1)*time.Hour),
+	)
+
+	s.queryAndDecode(f.TimeAdd(f.Epoch(0, f.TimeUnitSecond), 16, f.TimeUnitMinute), &t)
+	s.Require().Equal(t,
+		time.Unix(0, 0).UTC().
+			Add(time.Duration(16)*time.Minute),
+	)
+
+}
+
+func (s *ClientTestSuite) TestEvalTimeSubtractExpression() {
+	var t time.Time
+
+	s.queryAndDecode(f.TimeSubtract(f.Epoch(1, f.TimeUnitHour), 1, f.TimeUnitHour), &t)
+	s.Require().Equal(t, time.Unix(0, 0).UTC())
+
+	s.queryAndDecode(f.TimeSubtract(f.Epoch(16, f.TimeUnitSecond), 16, f.TimeUnitSecond), &t)
+	s.Require().Equal(t, time.Unix(0, 0).UTC())
```

Comment:
Could we change those to something that isn't zero? (just being picky, I know...)